src/manifestParser/manifestV2.ts

```diff
@@ -74,6 +78,19 @@ export default class ManifestV2 extends ManifestParser<Manifest> {
     return result;
   }
 
+  protected parseInputWebAccessibleScripts(
+    result: ParseResult<Manifest>
+  ): ParseResult<Manifest> {
+    result.manifest.web_accessible_resources?.forEach((scriptFile) => {
```

Comment:
This needs to be limited to files that match the defined script filter since it may contain non-script entries. Would also need to limit to pushing paths that are a single file since they can contain wildcard patterns which wouldn't be valid rollup inputs.